sdk/src/main/scala/com/horizen/forge/ForgerList.scala

```diff
@@ -0,0 +1,36 @@
+package com.horizen.forge
+
+import scorex.core.serialization.{BytesSerializable, ScorexSerializer}
+import scorex.util.serialization.{Reader, Writer}
+
+case class ForgerList(forgerIndexes: Array[Int]) extends BytesSerializable {
```

Comment:
case classes in Scala are considered to be immutable. So if we want to `updateIndexes` the best practice is to return the new `ForgerList` instance and keep original one as is:
`def updateIndexes(indexToUpdate: Array[Int]): ForgerList`